spec/requests/users_controller_spec.rb

```diff
@@ -106,13 +112,13 @@
     describe 'PATCH #update' do
       context 'with valid parameters' do
         let(:new_attributes) do
-          { password: 'devise_sucks' }
+          { email: 'devise_sucks@gmail.com' }
         end
 
         it 'updates the requested user' do
           patch user_url, params: { user: new_attributes }, headers: valid_headers, as: :json
           user.reload
-          expect(user.authenticate('devise_sucks')).to be_truthy
+          expect(user.unconfirmed_email).to equal('devise_sucks@gmail.com')
```

Comment:
Please re-add the password to the attributes and the expect line alongside the email ones